boards/arm/mimxrt1050_evk/pinmux.c

```diff
@@ -65,27 +74,63 @@ static int mimxrt1050_evk_init(struct device *dev)
 	IOMUXC_SetPinMux(IOMUXC_GPIO_AD_B0_03_LPSPI3_PCS0, 0);
 
 	IOMUXC_SetPinConfig(IOMUXC_GPIO_AD_B0_00_LPSPI3_SCK,
-			    IOMUXC_SW_PAD_CTL_PAD_PKE_MASK |
-			    IOMUXC_SW_PAD_CTL_PAD_SPEED(2) |
-			    IOMUXC_SW_PAD_CTL_PAD_DSE(6));
+				IOMUXC_SW_PAD_CTL_PAD_PKE_MASK |
```

Comment:
So, in such cases, Zephyr codestyle requires to align args "in column", using tabs for as long as possible, then spaces. In other words, what was before seems to be right. Also, IMHO, codestyle changes should be separated from functional changes. YMMV